charts/invenio/templates/install-init-job.yaml

```diff
@@ -56,10 +56,22 @@ spec:
             secretKeyRef:
               name: invenio-secrets
               key: INVENIO_CSRF_SECRET_SALT
+        volumeMounts:
+          {{- range $key, $value := .Values.invenio.vocabularies }}
+          - name: vocabularies
+            mountPath: "/opt/invenio/var/instance/app_data/vocabularies/{{ $key }}"
+            subPath: "{{ $key }}"
```

Comment:
Is `/opt/invenio/var/instance/app_data/vocabularies/` generally prepopulated by the image, so other files exists there without mount?

If so, we do need to use `subPath` like you do here, so the other files are not overwritten by the volume mount.

On the other hand, if that path is not prepopulated by the image, then I it would be advantageous to drop `subPath`. Mostly because `subPath` mounts are not updated without a pod restart, but also because that would let us drop the entire `range` since each field in a `ConfigMap` gets mounted then.

(same comment for the similar places in this PR).

Reply:
In general, the folder already existing inside the default docker image, https://github.com/inveniosoftware/cookiecutter-invenio-rdm/blob/master/%7B%7Bcookiecutter.project_shortname%7D%7D/Dockerfile#L20 and https://github.com/inveniosoftware/cookiecutter-invenio-rdm/tree/master/%7B%7Bcookiecutter.project_shortname%7D%7D/app_data/vocabularies.
I say _in general_ because one can always change the docker image and not add it ...

Reply:
It is more or less hardcoded here https://github.com/inveniosoftware/invenio-rdm-records/blob/master/invenio_rdm_records/fixtures/__init__.py#L23

Reply:
Thanks for checking. Then I think we must use `subPath`. Thus, this part looks good in its current form.